### ISSUE TITLE
onPage change doesnt detect page change

### ISSUE THREAD

**Issue by ronnizan** [react-pdf-viewer/react-pdf-viewer#1598]
i have a pdf with 3 pages, when scrolling the onPageChange event does not fire..
![image](https://github.com/react-pdf-viewer/react-pdf-viewer/assets/59262082/de934361-7388-4551-8c30-a1a381e207df)
i have a surrounding div with fixed height and width, and overflow-y: auto (also without the overflow its not working).

versions:
"@react-pdf-viewer/core": "^3.12.0",
 "pdfjs-dist": "3.4.120",

usage:
<Viewer
            fileUrl={fileUrl}
            onPageChange={({ currentPage }) => console.log(currentPage)}
...
 
thanks for the help!

**Comment by phuocng** [comment]
@ronnizan Does it happen with any document or just the specific 3-page document above?

**Comment by ronnizan** [comment]
@phuocng 
Yes, I tried 2 other documents, and the page change was not detected.

**Comment by adamjbragg** [comment]
I'm having the same issue.
